templates/404.html

```diff
@@ -5,10 +5,20 @@
 {% block content %}
 <div class="p-strip">
   <div class="row">
-    <div class="col-6">
-      <h1>404: Page not found</h1>
-      <p class="p-notification__response">
-        {{ error }}
+    <div class="col-10">
+      <h2>404: Aw, snap! We couldn't find that page</h2>
+      <h4>Looking for a snap? Try searching:</h4>
+      <form action="/search" class="p-form p-form--inline p-form--search">
+     <div class="p-form__group">
```

Comment:
Indentation sun!?

Reply:
What's the standard? Should I continue to indent subsequent items further in?

Reply:
All children should be indented. So every time you have an opening tag `<tag>` you should indent. Every time you have a closing tag `</tag>` you should outdent. You don't indent on elements that are self closing `<img />` `<input />` `<br />`